tests/fixtures/ecma-features/duplicateObjectLiteralProperties/proto-property.src.js

```diff
@@ -0,0 +1,8 @@
+"use strict";
+
+var proto = {};
+
+var x = {
+	__proto__: proto,
```

Comment:
Can you add another test using the string "__proto__" twice? It should not throw an error in that case.